### DISCUSSION TITLE
DietPi-Software | Glow Render markdown on the CLI, with pizzazz!

### DISCUSSION THREAD

**Discussion by anarkia47** [MichaIng/DietPi#6115] [Software Request]
#### Formal software information
- Software title | GLOW
- Short description | Glow is a terminal based markdown reader designed from the ground up to bring out the beauty—and power—of the CLI 
- Official URL (if available) | https://charm.sh
- GitHub URL (if available) | https://github.com/charmbracelet/glow
- (Official) install documentation | https://github.com/charmbracelet/glow

#### Are there similar/alternative software titles available with DietPi-Software?
Not that I'm aware 
- ...

#### What makes your requested software better than the above solutions, if available?
N/A
- ...

#### How can DietPi make the installation easier or compatible, than following the install instructions or do APT installation, if available?
<!-- EG: According to the official install instructions there, users need to install via snapd not sure if DietPi  supports this natively and wouldn't want to bork my installation of DietPi ...
- ...

#### Can you provide the installation steps that you would suggest DietPi-Software to do?
I'm not a dev, so I can't help you with that but from my basic understand it seems you have very little steps to do in order to make it work 

#### Are you willing to help maintaining the software installation, e.g. in case of needed setup changes due to updates etc.? 
As mentioned earlier, I'm not a dev, I'm just a hobbyist and thus the reason I'm asking for the implementation of this program, in my opinion this little tool will help a lot of newcomers access knowledge bases such as README.md etc much more accessible and with an easier reading layout

**Comment by MichaIng** [comment]
Many thanks for your request. Looks very interesting, the whole Charm toolset, probably even something among to use for DietPi tools in the future 😄.

I'll convert this into a discussion, so people can vote via ⬆️ button at your post.
